model-optimizer/extensions/middle/TFResizeNearestNeighborToInterpolateV4.py

```diff
@@ -0,0 +1,125 @@
+"""
+ Copyright (C) 2018-2020 Intel Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+import logging as log
+
+import numpy as np
+
+from extensions.ops.interpolate import Interpolate
+from mo.front.common.layout import get_height_dim, get_width_dim
+from mo.front.common.partial_infer.utils import int64_array
+from mo.front.tf.graph_utils import create_op_with_const_inputs
+from mo.graph.graph import Graph, Node, rename_nodes
+from mo.middle.replacement import MiddleReplacementPattern
+
+
+def replace_resize_nearest_neighbor(graph: Graph, resize: Node):
```

Comment:
Implementation of `replace_resize_nearest_neighbor` is very similar to `replace_resize_bilinear`. Can merge them into a single function?

Reply:
Done.